tests/integration/kw_ssh_test.sh

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env bash
+
+include './src/lib/kwio.sh'
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+
+# This function calls setup_nested_ssh_server to configure the SSH server
+# inside the container of each distribution in the DISTROS array.
+#
+# Returns:
+# return 0: If all SSH servers are configured successfully.
+# return 1: If the SSH server configuration fails for any distribution.
+function oneTimeSetUp()
+{
+  local distro
+  local container
+
+  for distro in "${DISTROS[@]}"; do
+    say "Configuring ssh server for ${distro}..."
```

Comment:
I think we can suppress "OK output" when everything is going as expected to not pollute the test suite. IMHO, a polluted test suite output has a feeling of fail. Makes sense?